src/covid19sim/hydra-configs/tracing-method/binary_digital_tracing_order_1.yaml

```diff
@@ -0,0 +1,10 @@
+---
+# NOTE: RISK_MODEL is only valid when INTERVENTION = "Tracing"
+RISK_MODEL : "digital" # "naive"  "manual", "digital", "transformer"
+SHOULD_MODIFY_BEHAVIOR : True
+
+## INTERVENTIONS
+INTERVENTION : "Tracing"
+
+# TRACING RISK MODEL PARAMETERS  (non-ML)
+TRACING_ORDER : 1
```

Comment:
I think leaving out other `Tracing` parameters makes it a bit confusing. I will prefer to have `TRACE_SYMPTOMS`, `TRACE_RISK` all in here. It will also lead to less confusion for the new comers.

Reply:
What do you think?

Reply:
my bias was to only put the configurations which will overwrite the base config settings into the tracing-mathod settings. I.e., minimal change for that exp in that file, otherwise default

Reply:
I don't have very strong feelings about this tho

Reply:
Another issue needs to be reopened to discuss folder structure!!!